import Notification from '../models/notification.model';
import handleError from './handleError';

export const pushNotification = async () => {
  try {
  } catch (error) {
    console.error(error);
  }
};
